feature-libs/customer-ticketing/assets/translations/en/customer-ticketing.ts

```diff
@@ -11,16 +11,19 @@ export const customerTicketing = {
     message: 'Message',
     cancel: 'Cancel',
     submit: 'Submit',
-    charactersLeft: 'Characters Left: {{count}}',
+    charactersLeft: 'characters left: {{count}}',
     uploadFile: 'Upload File',
-    fileSizeLimit: 'File Size Limit: {{count}}MB',
+    fileSizeLimit: 'File size limit: {{count}}MB',
     maximumAttachment: 'Maximum of one attachment per message',
     createNewTicket: 'Add',
     addNewRequest: 'Add New Request',
     subject: 'Subject',
     category: 'Category',
     associateTo: 'Associate To',
     attachments: 'Attachments',
+    ticketCreated: 'Request {{requestNo}} Created.',
```

Comment:
same as previous comment, let's do this for text that applies only for ticket creation

Reply:
Done